lib/octokit/client/organizations.rb

```diff
@@ -335,6 +335,22 @@ def team_by_name(org, team_slug, options = {})
         get "#{Organization.path(org)}/teams/#{team_slug}", options
       end
 
+      # Check team permissions for a repository
+      #
+      # Requires authenticated organization member.
+      #
+      # @param org [String, Integer] Organization GitHub login or id.
+      # @param team_slug [String, Integer] Team slug or Team ID.
+      # @param owner [String] Owner name for the repository.
+      # @param repo [String] Name of the repo to check permissions against.
+      # @return [Sawyer::Resource] Hash representing team permissions for the repository.
+      # @see https://docs.github.com/en/rest/teams/teams#check-team-permissions-for-a-repository
+      # @example
+      #   @client.team_permission_level("github", "justice-league", "octocat", "hello-world", :accept => 'application/vnd.github.v3.repository+json')
+      def team_permission_level(org, team_slug, owner, repo, options = {})
+        get "#{Organization.path(org)}/teams/#{team_slug}/repos/#{owner}/#{repo}", options
```

Comment:
It feels a bit weird that this method returns an empty string if the team _does_ have permissions on the repo. 

I wonder if we should use `#boolean_from_response` here to return a boolean instead, which feels like a nicer API for this Ruby method.

Another option that comes to mind is to have two versions of the method - one which sends the `Accept: application/vnd.github.v3.repository+json` header and gets a JSON response, and one which returns a boolean.